cinq_auditor_required_tags/__init__.py

```diff
@@ -326,30 +331,38 @@ def process_actions(self, actions):
         """
         notices = {}
         notification_contacts = {}
-
+        AWS_EC2_INSTANCE = 1
+        AWS_S3_BUCKET = 5
         try:
             for action in actions:
                 resource = action['resource']
-                if resource.resource_type == 'aws_ec2_instance':
-                    metrics = {"instance_type": resource.instance_type, "public_ip": resource.public_ip}
-
-                elif resource.resource_type == 'aws_s3_bucket':
-                    metrics = {"website_enabled": resource.website_enabled}
-
-                else:
-                    metrics = {}
+                metrics = {}
+                self.log.info('Resource is {} {}'.format(resource, dir(resource)))
+                if resource.resource_type_id == AWS_EC2_INSTANCE:
+                    for prop in resource.properties:
+                        if prop.name == "instance_type":
+                            instance_type = prop.value
+                        if prop.name == "public_ip":
+                            public_ip = prop.value
+                    metrics = {"instance_type": instance_type, "public_ip": public_ip}
+
+                elif resource.resource_type_id == AWS_S3_BUCKET:
+                    for prop in resource.properties:
+                        if prop.name == "metrics":
+                            metrics = prop.value
```

Comment:
Same kind of thing here.  It's possible that `metrics` is not going to be set and you reference further below.

Reply:
I'll set to 'unavailable'. Because 0 and 0 are valid values.